tekton-pipelines/base/tasks/slack-notification-task.yaml

```diff
@@ -106,4 +136,10 @@ spec:
         -field3_short true \
         -field4_title "Manage" \
         -field4_value "<$(params.repository_url)|Github> | <$(params.repository_url)/issues|Issues> | <$JIRA_PROJECT_URL|Jira> | <$TEKTON_URL/$pipelinerun_name|CI> | <$argo_app_url|CD> " \
-        -field4_short false
+        -field4_short false \
+        -field5_title "Failure info" \
+        -field5_value "$failed" \
+        -field5_short false \
+        -field6_title "Traceback" \
+        -field6_value "$error_msg" \
+        -field6_short false
```

Comment:
let's only send this WHEN we got actual error, so we don't have empty sections in "good" slack notification